Zend/Optimizer/zend_func_infos.h

```diff
@@ -312,6 +312,9 @@ static const func_info_t func_infos[] = {
 	F1("pg_host", MAY_BE_STRING),
 	F1("pg_version", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_LONG|MAY_BE_ARRAY_OF_STRING|MAY_BE_ARRAY_OF_NULL),
 	F1("pg_jit", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_STRING|MAY_BE_ARRAY_OF_NULL),
+#if defined(HAVE_PG_SERVICE)
```

Comment:
Did you add this by hand, or did you forget to update the optimizer infos after updating the annotation?

Reply:
I added after the first round of stub update forgot to rerun again ... should disappear